routes/web.php

```diff
@@ -18,16 +18,21 @@
 Auth::routes();
 
 Route::get('/home', 'HomeController@index')->name('home');
-Route::get('/users/{id}', 'UserController@show')->name('users.show');
+
 
 Route::get('login/{provider}/callback','SocialController@callback');
 Route::get('login/{provider}', 'SocialController@redirect');
 
-Route::group(['prefix'=>'/tags'],function() {
+Route::group(['prefix' => 'users'], function() {
+	Route::get('/{id}', 'UserController@edit')->name('users.edit');
+	Route::put('/{id}', 'UserController@update')->name('users.update');	
+});
+
+Route::group(['prefix' => 'tags'],function() {
```

Comment:
```suggestion
Route::group(['prefix' => 'tags'], function() {
```
Nếu e cài `phpcs` htrc a bảo, rồi chạy thì nó sẽ báo lỗi ngay nhé